Missing typings field in package.json

### Environment
Node version: 8.9.3 / 9
Npm version: 5.6.0 (or Yarn 1.5.1)
OS and version: Mac OS X 
vsts-node-api version: 6.5.0

### Issue Description
Since few versions ago, this package provides it's own `typings` instead of using `@types/vso-node-api` package. 
Since this PR: https://github.com/Microsoft/vsts-node-api/pull/154/ , this package no longer provides it's own typings (not sure why...) , and does not have the correct definition in `package.json`.
This causes build to fail with this error:
```
Try `npm install @types/vso-node-api` if it exists or add a new declaration (.d.ts) file containing `declare module 'vso-node-api';`
```

But `@types/vso-node-api` is deprecated now.


### Expected behaviour
The package should provide it's own typings because it's easier to maintain and keep up-to-date with the package's code.

### Actual behaviour
Error during build of a missing typings, and a deprecated typings package :(

### Steps to reproduce
1. Install `vso-node-api@6.5.0` in a TypeScript project, use it, and try to run `tsc`

### Logs
```
app.ts(1,55): error TS7016: Could not find a declaration file for module 'vso-node-api'. './node_modules/vso-node-api/WebApi.js' implicitly has an 'any' type.
  Try `npm install @types/vso-node-api` if it exists or add a new declaration (.d.ts) file containing `declare module 'vso-node-api';`
```

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "vso-node-api",
     "description": "Node client for Visual Studio Online/TFS REST APIs",
-    "version": "6.5.0",
+    "version": "6.5.1",
     "main": "./WebApi.js",
     "types" : "./WebApi.d.ts",
     "scripts": {

--- a/package.json
+++ b/package.json
@@ -3,6 +3,7 @@
     "description": "Node client for Visual Studio Online/TFS REST APIs",
     "version": "6.5.0",
     "main": "./WebApi.js",
+    "types" : "./WebApi.d.ts",
     "scripts": {
         "build": "node make.js build",
         "test": "node make.js test",

